recipe/meta.yaml

```diff
@@ -16,8 +16,10 @@ build:
     # d1trimfile option is added by a patch and disabled by unsetting SRC_DIR
     # https://github.com/conda-forge/python-feedstock/blob/e7ea437a819f8f06dcbc3769f6e46dc026070c44/recipe/patches/0021-Add-d1trimfile-SRC_DIR-to-make-pdbs-more-relocatable.patch
     - set "SRC_DIR="   # [win]
-    # unvendor xsimd; use version packaged by conda
-    - {{ PYTHON }} -c "import os, shutil; shutil.rmtree(os.path.join('pythran', 'xsimd'))"
+    # Set compilers to clang-cl
+    - set "CC=clang-cl.exe"  # [win]
+    - set "CXX=clang-cl.exe" # [win]
+    - set "LDFLAGS=" # [win]
```

Comment:
```suggestion
    - set "LDFLAGS=" # [win]
```
Is this intentionally left empty? We wouldn't use `none` in its place here?

Reply:
@cbouss I'd be interested to know about the second paragraph if you have the time